sample/rules.go

```diff
@@ -223,7 +223,9 @@ func ruleMatchesSpanInTrace(trace *types.Trace, rule *config.RulesBasedSamplerRu
 	return false
 }
 
-func extractValueFromSpan(trace *types.Trace, span *types.Span, condition *config.RulesBasedSamplerCondition, checkNestedFields bool) (interface{}, bool) {
+// extractValueFromSpan extracts from a span the value found at the first of the given condition's fields found on a span.
```

Comment:
"from a span"..."found on a span". This feels a bit vague. Is it the same span? Is it just duplicate info?

Reply:
This is the span that gets passed into the function. I'll fix the wording so its not ambiguous.